nucypher/blockchain/eth/clients.py

```diff
@@ -329,6 +342,27 @@ def sync(self, *args, **kwargs) -> bool:
         return True
 
 
+class InfuraClient(Web3Client):
+
+    is_local = False
+
+    def unlock_account(self, address, password):
+        return True
+
+    def sync(self, *args, **kwargs):
+        return True
+
+
+class InfuraClient(Web3Client):
```

Comment:
Did a merge cause this class to be defined twice?

Reply:
:heavy_check_mark: